Sources/Datadog/FeaturesIntegration/CITestIntegration.swift

```diff
@@ -0,0 +1,81 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+private enum DDCFMessageID {
+    static let customTags: Int32 = 0x1111
```

Comment:
Do we use it anywhere 🤔💭? If we need it, I think the name could be a bit more descriptive - maybe we can add some verb 🙂?

Reply:
The reason to keep the same enum used in the testing framework, but I can reduce it to only the needed values if you prefer. I will improve the description if we keep.

Reply:
We can keep it, no problem 👍. Good for parity between definitions in both SDKs 👌.

Reply:
I kept but changed the name to `setCustomTags`